Formula/notcurses.rb

```diff
@@ -0,0 +1,37 @@
+class Notcurses < Formula
+  desc "Blingful character graphics/TUI library"
+  homepage "https://nick-black.com/dankwiki/index.php/Notcurses"
+  url "https://github.com/dankamongmen/notcurses/archive/refs/tags/v2.3.12.tar.gz"
+  sha256 "ce042908fac11f7df1f9eaa610e46e9c615f53ab036b7c27ae2396292512407b"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "doctest" => :build
+  depends_on "pandoc" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ffmpeg"
+  depends_on "libunistring"
+  depends_on "ncurses"
+  depends_on "readline"
+  uses_from_macos "zlib"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args + %w[
+      -DBUILD_TESTING=on
+      -DBUILD_POC=off
+    ]
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    # without the redirect, it generates too much output, blowing up
+    # homebrew after much painful paging. without the TERM definition,
+    # Notcurses is like a small child, lost in a horrible mall.
+    # of course, if the tests are run in some non-xterm terminal, this
+    # choice might prove unfortunate.
+    # you have no chance to survive. make your time.
+    ENV["TERM"] = "xterm"
+    system "#{bin}/notcurses-tester > /dev/null"
```

Comment:
We can probably skip the redirect. You won't see the output anyway unless the test fails, or you ask for it with `--verbose`.

Reply:
fair enough! my only worry is that running `brew test --verbose notcurses` is then kind of a never-desirable thing, as all it's going to do is lock up the runner process in a highly nonobvious way. if they want to run it and see the output, they really just want to run a bare `notcurses-tester` process. just a comment.

Reply:
Done.